tests/check_merge_conflict_test.py

```diff
@@ -98,6 +114,13 @@ def repository_pending_merge(tmpdir):
         yield repo2
 
 
+def test_not_in_a_repository(tmpdir):
+    with tmpdir.as_cwd():
+        f1 = tmpdir.join('f1')
+        f1.write('foo\n')
+        assert main(['f1']) == 0
+
+
```

Comment:
I don't think this is a realistic test -- I expect these to always run inside a git repo